.gitignore

```diff
@@ -4,4 +4,5 @@ cmds/coredhcp/leases4.txt
 cmds/coredhcp/config.yml
 cmds/coredhcp/coredhcp
 cmds/client/client
+cmds/coredhcp-generator/coredhcp-generator
```

Comment:
I would simply call it generator or server generator, the coredhcp feels redundant.

Reply:
I don't have a strong opinion about it though.

Reply:
it was the original name, but if people do go install github.com/coredhcp/coredhcp/cmds/generator they will end up with a `generator` binary in their PATH. I used the longer name `coredhcp-generator` to remove ambiguity from the CLI name